test/config/application_config_test.rb

```diff
@@ -5,15 +5,18 @@
 # tests config/application.rb configuration to prevent the app from unexpected config changes!
 
 class ApplicationConfigTest < ActiveSupport::TestCase
+  setup do
+    @config = Rails.application.config
+  end
   test 'Configuration defaults should match with Rails version' do
-    assert_equal Rails.application.config.loaded_config_version, Rails.version.to_f
+    assert_equal @config.loaded_config_version, Rails.version.to_f
```

Comment:
`to_f` kullanmadan (diğer diff'te göründüğü gibi) Gem::Version'la daha legal yapabilirsin bunu.  Şu çalışmalı (untested):

```ruby
assert_equal Gem::Version.new(@config.loaded_config_version), Gem::Version.new(Rails.version)
```

FYI: Sürüm karşılaştırmalarında başka enteresan case'leri de görmek istersen: https://stackoverflow.com/questions/2051229/how-to-compare-versions-in-ruby

Reply:
Eskiden böyleydi zaten hocam. Bundaki sıkıntı `Gem::Version.new(@config.loaded_config_version)`'un `5.2` döndürüyor olması. Yani verdiğiniz örnek şöyle çalışıyor:

`assert_equal 5.2, 5.2.1`

Bu yüzden `to_f`'ler falan geldi koda :)